packages/fx-core/templates/plugins/resource/keyvault/bicep/keyVaultProvision.template.bicep

```diff
@@ -0,0 +1,54 @@
+@secure()
+param provisionParameters object
+param userAssignedIdentityObjectId string
+
+var resourceBaseName = provisionParameters.resourceBaseName
+var keyVaultName = contains(provisionParameters, 'keyVaultName') ? provisionParameters['keyVaultName'] : '${resourceBaseName}'
+var tenantId = subscription().tenantId
+var m365ClientSecretName = 'm365ClientSecret'
+var m365ClientSecret = provisionParameters['m365ClientSecret']
+var botClientSecretName = 'botClientSecret'
```

Comment:
do we need to review the secret names?

Reply:
@MuyangAmigo Please help check whether the name `botClientSecret` is OK.